preprocessing/filtered_innerouter_snsc_mousekidney.ipynb

```diff
@@ -210,90 +210,6 @@
     "bulk_adata = bulk_adata.transpose()"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    pca_X1 = pca_10.fit_transform(sn_adata.X) #sort of like the training dimensions
</pre><p>instead of X1 and X2 I would suggest to use more descriptive variable names</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/rnaseq_translator/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='130'/>